### PR TITLE
Questionnaire more devices

### PR DESCRIPTION
**Pull request by jsheppard95**
## Description
Made a new pull request with the correct commit history for my work on questionnaire-more-devices.

## Motivation and Context
Goal:
Allow happi program to load additional devices (currently just loads motors) from LCLS questionnaire and then instantiate as ophyd objects in beamline python.
This required changes to the actual questionnaire so that the output of:
`psdm_qs_cli.QuestionnaireClient.getProposalDetailsForRun` consistently returned a dictionary with CDS entries of the form:
`'pcdssetup-<device type>-<#>-<device type field>': '<Questionnaire Entry'>`
i.e:
`'pcdssetup-motors-14-name': 'nav_zoom'`

The class `happi.backends.qs_db.QSBackend` now loads motors, triggers, and ao/ai signals as shown with the additional entries in `happpi/tests/conftest.mockqsbackend`, but the questionnaire changes seem to have broken the following tests:

`happi.tests.test_backends.test_json_delete` 
`happi.tests.test_backends.test_json_save`
`happi.tests.test_client.test_create_valve[json]`, `happi.tests.test_client.test_add_and_find_device[json]`
`happi.tests.test_client.test_find_device[json]`
`happi.tests.test_client.test_remove_device[json]`

And so I will now focus on fixing these.

### CODE REVIEW

**Comment by jsheppard95** [comment]
@teddyrendahl I think this finally does what we wanted when you presented this task to me and at this point is ready for review. I'm not sure why the tests weren't passing before, but I just made sure everything was the same as pcdshub/happi/master except for the changes relevant to reading from the questionnaire. We can now load in motors, analog inputs/outputs, and triggers.

File: happi/tests/test_backends.py [hunk 1]
```diff
@@ -147,12 +147,13 @@ def test_json_initialize():
...
     assert len(mockqsbackend.find(name='sam_r', multiples=True)) == 1
 
 
 @requires_questionnaire
 def test_qsbackend_with_client(mockqsbackend):
     c = Client(database=mockqsbackend)
-    assert len(c.all_devices) == 6
-    assert all([isinstance(d, Motor) for d in c.all_devices])
+    assert len(c.all_devices) == 14
+    assert all([isinstance(d, Motor) or isinstance(d, Trigger)
```
Comment (teddyrendahl) [inline happi/tests/test_backends.py]: You should check you got the proper number of each i.e
```python
device_types = [device.__class__ for device in c.all_devices]
assert device_types.count(Trigger) == 4
...
```

File: happi/tests/test_client.py [hunk 2]
```diff
@@ -82,6 +82,7 @@ def test_create_valve(happi_client, valve_info):
     assert device.beamline == valve_info['beamline']
     # Save
     device.save()
+    # seeing some format differences between entries in testing.json
```
Comment (teddyrendahl) [inline happi/tests/test_client.py]: Still need this comment?

File: happi/containers.py [hunk 3]
```diff
@@ -297,3 +297,33 @@ class Motor(Device):
...
+    system = copy(Device.system)
+    system.default = 'camera'
+
+
+class Acromag(Device):
+    """
+    A Generic class for Acromag
+    """
+    device_class = copy(Device.device_class)
+    device_class.default = 'pcdsdevices.device_types.Acromag'
```
Comment (teddyrendahl) [inline happi/containers.py]: Does this class exist?

Comment (jsheppard95) [reply happi/containers.py]: It does with this PR: https://github.com/pcdshub/pcdsdevices/pull/329

File: happi/tests/test_client.py (hunk 2 above)
Comment (jsheppard95) [reply happi/tests/test_client.py]: No and my bad forgot I should be using the logger for messages like these - then won't have to remember to delete.

File: happi/tests/test_backends.py (hunk 1 above)
Comment (jsheppard95) [reply happi/tests/test_backends.py]: Good call, added this in 08d8c41

File: happi/containers.py [hunk 4]
```diff
@@ -297,3 +297,33 @@ class Motor(Device):
...
+
+
+class Trigger(Device):
+    """
+    A Generic class for Controls Triggers
+    """
+    device_class = copy(Device.device_class)
+    device_class.default = 'pcdsdevices.device_types.Trigger'
+    system = copy(Device.system)
+    system.default = 'trigger'
```
[the remark this answers is not in the document]
Comment (jsheppard95) [reply happi/containers.py]: Changed this as well in 08d8c41

**Comment by jsheppard95** [comment]
Closes #86

**Pull request merged**